halyard-core/src/main/java/com/netflix/spinnaker/halyard/core/job/v1/TeeByteArrayOutputStream.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2017 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.halyard.core.job.v1;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+
+public class TeeByteArrayOutputStream extends ByteArrayOutputStream {
```

Comment:
Are you able to use this instead of writing this from scratch?

https://commons.apache.org/proper/commons-io/javadocs/api-1.4/org/apache/commons/io/output/TeeOutputStream.html